Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json

```diff
@@ -180,7 +180,16 @@
                                 "index": 10,
                                 "sectionItemType": "field",
                                 "startCol": 0
-                            }
+                            },
+							{
+								"endCol": 2,
+								"fieldId": "categories",
+								"height": 22,
+								"id": "5e2287d0-e502-11ed-ba0f-99a713ead7dc",
+								"index": 11,
+								"sectionItemType": "field",
+								"startCol": 0
+							}
```

Comment:
Make it align and on the same line like the rest of the fields

Reply:
![Screenshot 2023-05-01 at 11 42 27](https://user-images.githubusercontent.com/93524335/235430365-67a65e80-d698-434a-aee2-7d13c7a20008.png)
In the code itself it looks ok, maybe the display in github making it look weird